Ignore Proton Experimental and Wallpaper Engine

SpyderCoder opened the pull request:
I added a couple things to the ignore list.

mmatyas commented:
Thanks! I'm fine with ignoring Proton, but I think Wallpaper Engine is a valid entry to launch. In general, things you can launch on their own from Steam should also be visible in Pegasus too.

SpyderCoder commented:
Ah ok I wasn't too sure. If you look at the new commit I've removed Wallpaper Engine and I've added the remaining Proton and Steam Linux Runtime versions. I guess I'll start looking into building a hide application feature.

mmatyas commented:
Thanks!
